packages/x-components/src/components/modals/__tests__/base-events-modal.spec.ts

```diff
@@ -44,7 +44,7 @@ function mountBaseEventsModal({
       await wrapper.find(getDataTestSelector('modal-overlay'))?.trigger('click')
     },
     async emit(event) {
-      await XPlugin.bus.emit(event)
+      await XPlugin.bus.emit(event, undefined)
```

Comment:
Vamos a revisar los tipos del bus, me suena que habia dejado un comentario en https://github.com/empathyco/x/pull/1743 pero no lo encuentro, igual se perdio o al final no puse nada.

Es decir, en vez de cambiar todos los usos, revisar y arreglar los tipos

Reply:
Encontrado https://github.com/empathyco/x/pull/1743/commits/ad32dd0e69a3cb41629379948e7d465907efc43a#r2037223869

Reply:
As we already commented, we go forward to make stable the repo again and then we can work in the trade-offs